bench/lib/benchmark.js

```diff
@@ -11,8 +12,20 @@ export type Measurement = {
     time: number
 };
 
+type Location = {
+    description: string,
+    tileID: Array<OverscaledTileID>,
+    zoom: number,
+    center: LngLatLike
+};
+
 class Benchmark {
-    constructor() {
+    styleURL: string;
+    locations: ?Location;
+
+    constructor(styleURL: string, locations: ?Location) {
```

Comment:
I'm confused by the plurality/singularity of `locations: ?Location`. The type and the way it's used in paint/layout makes it seem like this should be a singular "location" member. Where it's plural (non-paint/layout), it seems like the type would actually be `Array<Location>` (although I guess there's also the separate case of one "location" with multiple tiles, that doesn't look like what gets returned from `import { locations } from './lib/style_locations';`

Reply:
I would guess that it's easy to get flow errors because if it can be singular or plural that has to be explicitly checked all the time before you work with it. If it can be one or more, standardizing on plural seems good -- probably "plural but only one item" doesn't need to be handled as a special case?

Reply:
By "standardizing on plural", do you mean making `this.locations` always be an array of Location objects, even if it's just an array with a length of 1? That seems like the most logical way to square this circle in my brain.

Reply:
Yup. :+1: